src/model_config_utils.cc

```diff
@@ -1620,6 +1653,17 @@ ValidateModelConfig(
             " cache.");
   }
 
+  // If model_metric is specified, validate it.
+  if (config.has_model_metrics()) {
+#ifdef TRITON_ENABLE_METRICS
+    Status status = ValidateModelMetrics(config.model_metrics());
```

Comment:
There should be some macro for this like RETURN_IF_ERR

Reply:
Fixed